### PR TITLE
Add control & listen URLs to netmaster

### PR DESCRIPTION
**Pull request by neelimamukiri**
build.sh changes are temporary till the corresponding ansible & netplugin changes are merged.
ansible changes: https://github.com/contiv/ansible/pull/342/commits/2895ebde4ad9aca7e6e8445d85e2987dc47035d8
netplugin changes: https://github.com/contiv/netplugin/pull/835

### CODE REVIEW

**Comment by rhim** [comment]
@neelimamukiri also requires doc changes to http://contiv.github.io/documents/reference/netctlcli.html

File: install/ansible/env.json [hunk 1]
```diff
@@ -12,5 +12,7 @@
   "docker_reset_image_state": "False",
   "etcd_cleanup_state": "False",
   "auth_proxy_local_install": "False",
+  "listen_url": ":9999",
+  "control_url": ":9999",
```
Comment (gaurav-dalvi) [inline install/ansible/env.json]: both ports should be the same ? or for setup we are making it as same ?

File: scripts/build.sh [hunk 2]
```diff
@@ -86,7 +86,7 @@ curl -sSL https://github.com/contiv/netplugin/releases/download/$contiv_version/
 pushd $output_dir
 tar oxf netplugin-$contiv_version.tar.bz2 netctl
 rm -f netplugin-$contiv_version.tar.bz2
-git clone http://github.com/contiv/ansible
+git clone http://github.com/neelimamukiri/ansible -b netmaster_args
```
Comment (gaurav-dalvi) [inline scripts/build.sh]: do we need to change this ?

Comment (dseevr) [inline scripts/build.sh]: Looks like this was a local test of the changes on the ansible repo... this can be removed

**Comment by unclejack** [comment]
@neelimamukiri: It seems this needs a rebase.

**Comment by DivyaVavili** [comment]
@neelimamukiri: Could you rebase and update this? Can we target to merge this for 1.1 release?

**Pull request closed without merging**